otel/conf.go

```diff
@@ -421,11 +421,16 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
         action: update
         new_name: processes/cpu_time
         operations:
+          - action: experimental_scale_value
```

Comment:
Does this need a comment? E.g., "scale from seconds to microseconds"?

Reply:
done